doc-examples/jdbc-example-java/src/main/java/example/ContactView.java

```diff
@@ -0,0 +1,64 @@
+package io.micronaut.data.tck.entities;
```

Comment:
package should be example and some classes are also missing here
Why are classed copied here if there are no tests using them?